WebApplication/Backend/Repositorys/FeedbackRepository.cs

```diff
@@ -16,7 +16,8 @@ public FeedbackRepository()
         {
             try
             {
-                connection = new MySqlConnection("server=localhost;port=3306;database=mydb;user=root;password=root");
+                connection = new MySqlConnection("server=localhost;database=newdb;user=root;password=root");
+                connection.Open();
```

Comment:
I think only one is enough hahaha

Reply:
And I don't think it should always be open? But only when you need to make a request? So in the method that gets executes queries